app/models/game.rb

```diff
@@ -8,6 +8,18 @@ class Game < ActiveRecord::Base
 
   after_create :populate_board!
 
+  def determine_check
```

Comment:
I think a better name for this would be `check?` or `in_check?`. What do you think?